### PR TITLE
Pioneer 13a Gliders

### PR DESCRIPTION
**Pull request by cdobs**
Ingestion csvs for recovered data from gliders deployed on P13a:
-CP05MOAS-GL340 R00010
-CP05MOAS-GL380 R00008
-CP05MAOS-GL388 R00009

### CODE REVIEW

File: CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv [hunk 1]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/merged/cp_*.mrg,CP05MOAS-GL340-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/dvl/*.PD0,CP05MOAS-GL340-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/merged/cp_*.mrg,CP05MOAS-GL340-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/merged/cp_*.mrg,CP05MOAS-GL340-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/merged/cp_*.mrg,CP05MOAS-GL340-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL340/R00010/merged/cp_*.mrg,CP05MOAS-GL340-05-PARADM000,recovered_host,Available,
```
Comment (pbrickley) [inline CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv:7]: A OK

File: CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv [hunk 2]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/merged/cp_*.full.mrg,CP05MOAS-GL380-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/dvl/*.PD0,CP05MOAS-GL380-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/merged/cp_*.full.mrg,CP05MOAS-GL380-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/merged/cp_*.full.mrg,CP05MOAS-GL380-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/merged/cp_*.full.mrg,CP05MOAS-GL380-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL380/R00008/merged/cp_*.full.mrg,CP05MOAS-GL380-05-PARADM000,recovered_host,Available,
```
Comment (pbrickley) [inline CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv:7]: OK

File: CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv [hunk 3]
```diff
@@ -0,0 +1,7 @@
+parser,filename_mask,reference_designator,data_source,status,notes
+mi.dataset.driver.moas.gl.engineering.glider_eng_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/merged/cp_*.mrg,CP05MOAS-GL388-00-ENG000000,recovered_host,Available,
+mi.dataset.driver.moas.gl.adcpa.adcpa_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/dvl/*.PD0,CP05MOAS-GL388-01-ADCPAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.flort_m.flort_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/merged/cp_*.mrg,CP05MOAS-GL388-02-FLORTM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.ctdgv.ctdgv_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/merged/cp_*.mrg,CP05MOAS-GL388-03-CTDGVM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.dosta.dosta_abcdjm_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/merged/cp_*.mrg,CP05MOAS-GL388-04-DOSTAM000,recovered_host,Available,
+mi.dataset.driver.moas.gl.parad.parad_m_glider_recovered_driver,/omc_data/whoi/OMC/CP05MOAS-GL388/R00009/merged/cp_*.mrg,CP05MOAS-GL388-05-PARADM000,recovered_host,Available,
```
Comment (pbrickley) [inline CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv:7]: Very good

File: CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv (hunk 1 above)
Comment (kuo-james) [reply CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv:7]: Looks good.

File: CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv (hunk 2 above)
Comment (kuo-james) [reply CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv:7]: Under filename_mask: /cp_*.full.mrg vs /cp_380*.full.mrg

File: CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv (hunk 1 above)
Comment (johnmlund) [inline CP05MOAS-GL340/CP05MOAS-GL340_R00010_ingest.csv:7]: Looks good.
James and John

File: CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv (hunk 3 above)
Comment (kuo-james) [reply CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv:7]: Good

File: CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv (hunk 2 above)
Comment (johnmlund) [reply CP05MOAS-GL380/CP05MOAS-GL380_R00008_ingest.csv:7]: looks good

File: CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv (hunk 3 above)
Comment (johnmlund) [reply CP05MOAS-GL388/CP05MOAS-GL388_R00009_ingest.csv:7]: ok

**Pull request merged**